### PR TITLE
Output utilization fraction for solution.

### PR DESCRIPTION
**Pull request by seatim**

### CODE REVIEW

**Review by mcfarljm — APPROVED** [review]
LGTM

File: crop_rotation.py [hunk 1]
```diff
@@ -388,6 +388,19 @@ def render_solution(self, path):
         fig.savefig(path)
         print(f'Saved illustration of solution to {path}')
 
+    @property
+    def utilization(self):
+        """Return the fraction of the maximum possible plot utilization,
+        effected by the found solution.
```
Comment (JoelPasvolsky) [inline crop_rotation.py:394]: I find this a bit hard to parse: because of the comma, the antecedent of "effected by" is "Return" rather than "fraction". The first clause stands fine alone. I think the second clause is just to highlight that the fraction is calculated for the found solution -- I think that would be clear without the clause but if you feel a strong need to emphasize it, perhaps something like "Return the fraction of the maximum possible plot utilization for the found solution"? (Borderline but the alternative might be to state the obvious in a second sentence)

**Review by JoelPasvolsky — APPROVED** [review]

File: crop_rotation.py (hunk 1 above)
Comment (seatim) [reply crop_rotation.py:394]: 👍 Thanks, @JoelPasvolsky .  I like that alternative and will make that change.

**Review by hhtong — APPROVED** [review]
